ext/azuredevops/setupAzd/task.json

```diff
@@ -0,0 +1,30 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Microsoft/azure-pipelines-task-lib/master/tasks.schema.json",
+    "id": "854fea0d-8738-413e-ad4d-600113529a20",
+    "name": "setup-azd",
+    "friendlyName": "Setup Azure Developer CLI",
+    "description": "Install azd",
+    "helpMarkDown": "",
+    "category": "Utility",
+    "author": "AzureDeveloperCLI",
+    "version": {
+        "Major": 0,
+        "Minor": 12,
+        "Patch": 0
```

Comment:
Is our intent to have the rev at the same cadence as the CLI?

Reply:
No, we release on demand.